### PR TITLE
Release/1.9 openet prasanth

### PR DESCRIPTION
**Pull request by prasanth-openet**
# Description

Please include a summary of the changes.

This bug fixes ... \ This new feature can be used to ...

# Context

Users ran into this bug doing ... \ Users needed this feature to ...

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [ ] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by rbsrafa** [comment]
Hi @prasanth-openet , we need to change the chart version on this file: https://github.com/openet/gloo/blob/release/1.9-openet-prasanth/install/helm/gloo/generate.go#L110

After changing it could you re-run all the tests in Jenkins please? Thanks.

**Review by rbsrafa — DISMISSED** [review]
Hi @prasanth-openet  , we need to change the chart version on this file: https://github.com/openet/gloo/blob/release/1.9-openet-prasanth/install/helm/gloo/generate.go#L110

After changing it could you re-run all the tests in Jenkins please? Thanks.

**Comment by prasanth-openet** [comment]
> Hi @prasanth-openet , we need to change the chart version on this file: https://github.com/openet/gloo/blob/release/1.9-openet-prasanth/install/helm/gloo/generate.go#L110
> 
> After changing it could you re-run all the tests in Jenkins please? Thanks.

yes it is done. thanks

**Comment by prasanth-openet** [comment]
> Hi @prasanth-openet , we need to change the chart version on this file: https://github.com/openet/gloo/blob/release/1.9-openet-prasanth/install/helm/gloo/generate.go#L110
> 
> After changing it could you re-run all the tests in Jenkins please? Thanks.

Fixed

File: install/helm/gloo/generate.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func generateChartYaml(version string) error {
 		return err
 	}
 
-	chart.Version = "1.9.26-OPENET"
+	chart.Version = "1.9.30-OPENET-SNAPSHOT"
```
Comment (tiberiuac) [inline install/helm/gloo/generate.go]: Hi @prasanth-openet . All good, I think now you should hardcode the version to 1.9.30-OPENET, and then you can merge it to the target branch. Then, create a release tag with 1.9.30-OPENET

Comment (prasanth-openet) [reply install/helm/gloo/generate.go]: I have removed it now. Going to merge

**Pull request merged**